### PR TITLE
Update support ticket priorities to reflect Jira

### PR DESCRIPTION
**Pull request by karl-johan-grahn**
Update ticket priorities

### CODE REVIEW

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-78-6b21a419`

File: docs/content/sre/support/support.md [hunk 1]
```diff
@@ -35,25 +35,27 @@ As a registered user you can go to the [Service Desk Portal](https://stakater-cl
 
 ## Urgency Levels
```
Comment (rasheedamir) [inline docs/content/sre/support/support.md]: @karl-johan-grahn should we change this to priority as well?

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-78-06c2b451`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-78-0eee5b8e`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:v0.0.53`

**Pull request merged**